src/BreadcrumbItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import SafeAnchor from './SafeAnchor';
+
+const BreadcrumbItem = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    linkId: React.PropTypes.string,
+    href: React.PropTypes.string,
+    title: React.PropTypes.node,
+    target: React.PropTypes.string
+  },
+
+  render() {
+    const {
+      href,
+      title,
+      target,
+      linkId,
+      children,
+      ...props } = this.props;
+    const active = (href == null);
```

Comment:
On second thought, that's really just overthinking it. If the user wants additional styling on the rendered child, they can pass in a React node instead of a string.

Why don't you just call `warning` if both `href` and `active` are specified, but otherwise keep things as-is?